docs/sql-reference/sql-statements/data-definition/CREATE TABLE.md

```diff
@@ -503,6 +503,16 @@ PROPERTIES (
 | dynamic_partition.prefix    | No       | The prefix added to the names of dynamic partitions. Default value: `p`. |
 | dynamic_partition.buckets   | No       | The number of buckets per dynamic partition. The default value is the same as the number of buckets determined by the reserved word `BUCKETS` or automatically set by StarRocks. |
 
+#### Specify the bucket size for tables configured with random bucketing
+
+Since v3.2, for tables configured with random bucketing, you can specify the bucket size in the `PROPERTIES` using the `bucket_size` parameter during table creation. The default size is `1024 * 1024 * 1024 B` (1 GB), and the maximum size is 4 GB.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3308d1da-f866-40e8-8b9b-56aefa194919)
-> you can specify the bucket size by using the `bucket_size` parameter in `PROPERTIES` at table creation